source/amazon/services/prerequisites/considerations.rst

```diff
@@ -222,3 +222,7 @@ The following is an example of a valid configuration.
     </service>
 
   </wodle>
+  
+.. note::
+   Once the configuration is complete, you can enable the `Security Information` visualization feature in the **Wazuh Dashboard**. You can read more about it `here <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_.
+
```

Comment:
The note has little visibility inside of **FIPS endpoints**. I suggest adding a new section with this content.